docs/user-guide/dream/dream-advanced-powder-reduction.ipynb

```diff
@@ -453,7 +461,134 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "result.hist(tof=300).plot()"
+    "histograms = []\n",
+    "for events in result.values():\n",
+    "    hist = events.hist()\n",
+    "    # Apply the mask\n",
+    "    if (mask := hist.masks.pop('zero_vanadium', None)) is not None:\n",
+    "        hist.data = sc.where(mask, sc.scalar(0.0), hist.data)\n",
+    "    histograms.append(hist)\n",
+    "\n",
+    "sc.concat(histograms, dim='detector').sum('detector').plot()"
```

Comment:
I can't figure out in my head if this is a valid thing to do.
To me, the plot below where we first convert to d-spacing and 2theta, and then sum over 2theta makes sense.
This one does something different (the plots look different at least).

I guess I'm not sure why they are different...

Reply:
They are normalised differently. This one here computes an I(d) curve for both sample and vanadium and divides the two. The other case computes an I(2theta, d) curve for sample and vanadium, divides them, and only then removes the 2theta dimension.

Reply:
From today's discussion, it sounds like we need to compute 2d maps (2theta - dspacing) for sample and vanadium, divide them, and then sum along 2theta?

Reply:
I took that away from it, too. But I don't like introducing an arbitrary binning in 2theta in an intermediate step. (When we produce 1D outputs.)

Reply:
Instead of having something arbitrary, you could use the mean pixel size and mean distance to compute a theta resolution?